test/spec/adapters/sekindoUM_spec.js

```diff
@@ -0,0 +1,88 @@
+import {expect} from 'chai';
+import _ from 'lodash';
+var utils = require('../../../src/utils');
```

Comment:
`utils` doens't seem to be used in this file either and could also be removed if unneeded